messaging/index.js

```diff
@@ -87,12 +86,17 @@ export class Messaging {
             method: name,
             params: data,
         });
-        this.transport.notify(message);
+        try {
+            this.transport.notify(message);
+        } catch (e) {
+            // Silently ignoring any transport errors here as per section 4.1 of https://www.jsonrpc.org/specification
+            // Notifications are fire+forget and should be able to be sent without any knowledge of the receiving ends support
+        }
```

Comment:
This was already the case on other platforms, just on webkit we were bubbling up an error which we should not have.